src/set_theory/ordinal_arithmetic.lean

```diff
@@ -2078,6 +2104,9 @@ theorem is_normal.le_nfp {f} (H : is_normal f) {a b} :
 theorem nfp_eq_self {f : ordinal → ordinal} {a} (h : f a = a) : nfp f a = a :=
 le_antisymm (sup_le.mpr $ λ i, by rw [iterate_fixed h]) (le_nfp_self f a)
 
+protected lemma monotone.nfp {f : ordinal → ordinal} (hf : monotone f) : monotone (nfp f) :=
```

Comment:
Don't you wanna pull out `f` to the closest `variables`?

Reply:
What do you mean?

Reply:
It appears in most statements, so it could be declared via `variables`.